src/routes/api/payments/updateMember/+server.ts

```diff
@@ -37,18 +37,22 @@ ${MEMBER_API_URL}/organization/${MEMBER_ORGANIZATION}/members\
     throw error(500, response.statusText);
   }
   const existingMembers: Member[] = await response.json();
-  let newMembers: Member[] = people.map((person) => ({
-    name: person.name,
-    attributes: {
-      "Last Name": person.name.split(" ").pop() ?? person.name,
-      "House Number": houseNumber,
-      Street: streetName,
-      Neighborhood: neighborhood,
-      "Phone Numbers": [person.phone],
-      Emails: [person.email],
-      "Years Paid": [new Date().getFullYear().toString()],
-    },
-  }));
+  let newMembers: Member[] = people.map((person) => {
+    const trimmedName = person.name.trim();
+    return {
+      name: trimmedName,
+      attributes: {
+        "Last Name": trimmedName.split(" ").pop() ?? "",
+        "House Number": houseNumber,
+        Street: streetName,
+        Neighborhood: neighborhood,
+        "Phone Numbers": [person.phone],
+        Emails: [person.email],
+        "Years Paid": [new Date().getFullYear().toString()],
+        "Full Address": `${houseNumber} SW ${streetName}`,
```

Comment:
Is this like just logic specific to the neighborhood? All streets start with SW?

Reply:
All the numbered ones yes but some are named actually and I need to fix that, good call